### DISCUSSION TITLE
Questions about Basin Boundary Types

### DISCUSSION THREAD

**Discussion by Jinbow-0217** [SynxFlow/SynxFlow#22] [Q&A] [answered]
Hi, Dr.Xia:
In the flood simulation code, I observed three different types: rigid, open, and fall. What do each of these represent?

**Comment by xiaxilin** [comment]
Rigid means 'wall boundary', Open means free connection to water body or river inflow, Fall means water frees run out but no water flows into the domain.

**Reply by Jinbow-0217** [reply, marked as answer]
Thank you, Dr. Xia, for your answer. It cleared up my confusion.
